template/_eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "root": true,
+
+  "extends": "@react-native-community",
+  
+  "env": {
+    "jest": true
```

Comment:
I suggest something like this:

```js
  overrides: {
    files: ['**/__tests__/**/*.js', '**/?(*.)(spec|test).js'],
    env: {
      jest: true,
      'jest/globals': true,
    },
  },
```

Reply:
Thanks @satya164, I think it's a good point.

Reply:
Maybe we should add it to community config itself?

Reply:
I think keeping eslint test runner configuration within the project is ok. Jest is integrated into RN template, but other projects might depend on a different test runner. I'm fine with both solutions tho.

Reply:
@michalchudziak can we put it into the eslint config? I think that's better, especially since people can create a project without Jest still and it makes this a bit confusing.